contents/docs/user-guides/recordings.mdx

```diff
@@ -34,10 +33,28 @@ To do so, you should add the CSS class name `ph-no-capture` to elements which sh
 
 Additionally, when dealing with inputs, if you wish to still capture the input box but not its contents, you can use the class name `ph-ignore-input` instead.
 
-## Controlling the volume of recordings
+## Recordings data retention
+
+Depending on how your app/website is built, recordings can take a lot of disk space. To manage this, we have the following retention policy options in place.
+
+### PostHog Cloud & Clickhouse Self-hosted
+
+If you're running a Clickhouse instance, recordings are **automatically deleted after 3 weeks**. Old recordings are deleted with Clickhouse's <a target="_blank" href="https://clickhouse.com/docs/en/sql-reference/statements/alter/ttl/">Table TTL</a>. You can set your own retention policy by executing a query like this on your Clickhouse instance.
+
+```
+ALTER TABLE session_recording_events TTL toDate(created_at) + INTERVAL 6 WEEK
+```
+
+### Legacy Postgres Self-hosted
+
+Recordings are kept indefinitely by default. You can can set recordings to delete after a configurable number of days in the project settings page (/project/settings#recordings).
```

Comment:
Looks like we're missing the square brackets around 'project settings page' for the link.

Reply:
Deliberate, because these users are self-hosted, so we won't know their domain

Reply:
O I was just talking about the markdown syntax for the link. Added a commit to fix.